File: Flow Control/C++/flow.cpp
#include <iostream>

int main() {
    int x = 10;
    int arr[5] = {1,2,3,4,5};

    // If statement
    if (x > 0){
        std::cout << "Condition met.." << std::endl;
    } else {
        std::cout << "Condition not met.." << std::endl;
    }

    // For loop
    std::cout << "For loop:" << std::endl;
    for (int i=0; i < 5; i++){
        std::cout << arr[i];
    }
    std::cout << std::endl;

    // While loop
    std::cout << "While loop:" << std::endl;
    int iterator = 0;
    while (iterator < 5) {
        std::cout << iterator << " ";
        iterator++;
    }
    std::cout << std::endl;

    // Switch case
    switch (x) {
        case 0:
            std::cout << "X = 0";
            break;
        case 10:
            std::cout << "X = 10";
            break;
        default:
            std::cout << "X is not equal to neither 0 or 10";
    }

    return 0;
}

File: Flow Control/Go/flow.go
package main

import "fmt"

func main(){
	var x int = 10
	var arr [5]int = [5]int{1,2,3,4,5}

	if x > 0 {
		fmt.Println("X > 0")
	} else {
		fmt.Println("X <= 0")
	}

	fmt.Println("For loop:")
	for i:=0; i < 5; i++ {
		fmt.Print(arr[i])
	}
	fmt.Println(" ")

	fmt.Println("While loop:")
	iterator := 0
	for iterator < 5 {
		fmt.Print(iterator)
		iterator++
	}
	fmt.Println(" ")

	switch x {
	case 0:
		fmt.Println("X = 0")
	case 10:
		fmt.Println("X = 10")
	default:
		fmt.Println("X is not equal to neither 0 or 10")
	}
}